resources/custom.Makefile

```diff
@@ -1,10 +1,24 @@
 # This includes all the preprocessing not directly related to the PhEval Running
 
-PHENOTYPE_VERSIONS				:=	2309 2402
-
+PHENOTYPE_VERSIONS		:=	2309 2402
+EXOMISER_VERSIONS		:=	13.3.0 14.0.0
+PHEVAL_ZENODO_DATA_URL	:=	https://zenodo.org/records/11458312/files/monarch_pheval.tar.gz
 
 .PHONY: setup
-setup: download-phenotype
+
+setup: $(TMP_DATA)/monarch_pheval/Makefile
+
+$(TMP_DATA)/monarch_pheval.tar.gz:
+	mkdir -p $(TMP_DATA)
+	wget $(PHEVAL_ZENODO_DATA_URL) -O $@
+
+
+$(TMP_DATA)/monarch_pheval/Makefile: $(TMP_DATA)/monarch_pheval.tar.gz
```

Comment:
Why the name of the target? does $(TMP_DATA)/monarch_pheval/Makefile exist after the extraction?

If so, end the goal with 

```
test -f $(TMP_DATA)/monarch_pheval/Makefile
```

to ensure this file actually exists.

Reply:
The aim here is to prevent any changes made in the Makefile before the `make all` execution is replaced during the tarball extraction.